### PR TITLE
E0d/fix insights

### PR DESCRIPTION
**Pull request by e0d**
@cpennington One more.

### CODE REVIEW

File: docker/build/insights/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM edxops/precise-common:latest
+MAINTAINER edxops
+
+RUN apt-get update
```
Comment (cpennington) [inline docker/build/insights/Dockerfile]: Shouldn't this be part of the ansible commands?

Comment (e0d) [inline docker/build/insights/Dockerfile]: Yeah, this can be removed from all the Dockerfiles after a prior dependency re-org.

**Comment by e0d** [comment]
rereview @cpennington

File: docker/build/insights/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM edxops/precise-common:latest
+MAINTAINER edxops
+
+ADD . /edx/app/edx_ansible/edx_ansible
+COPY docker/build/insights/ansible_overrides.yml /
+WORKDIR /edx/app/edx_ansible/edx_ansible/docker/plays
+RUN /edx/app/edx_ansible/venvs/edx_ansible/bin/ansible-playbook insights.yml -i '127.0.0.1,' -c local -t "install:base,install:system-requirements,install:configuration,install:app-requirements,install:code" -e@/ansible_overrides.yml
```
Comment (cpennington) [inline docker/build/insights/Dockerfile]: Nit: How would you feel about breaking this over multiple lines? It's a bit hard on the eyes at the moment.

**Comment by cpennington** [comment]
:+1:

**Pull request merged**